src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobContainerInfoContributorTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.controllers;
+
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.models.BlobContainerItem;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationTest;
+
+import java.util.stream.Stream;
+
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@AutoConfigureMockMvc
+@IntegrationTest
+public class BlobContainerInfoContributorTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private BlobServiceClient blobServiceClient;
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void should_return_info_about_application() throws Exception {
+
+        PagedIterable pagedIterable = mock(PagedIterable.class);
+        given(blobServiceClient.listBlobContainers()).willReturn(pagedIterable);
+        given(pagedIterable.stream())
+            .willReturn(Stream.of(
+                new BlobContainerItem().setName("sscs"),
+                new BlobContainerItem().setName("bulkscan"),
+                new BlobContainerItem().setName("sscs-rejected")
+                )
+            );
+
+        this.mockMvc.perform(get("/info"))
+            .andDo(print())
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.app.name").isNotEmpty())
```

Comment:
we want to be sure of the app name. it plays significant role in app insights cloud role

Reply:
although it's not part of class name test but it is part of application test

Reply:
these did not make sense for me 
`        "name": "Bulk Scanning",
        "description": "Ingest scanned documents."
`

Reply:
oh dear. this needs fixing:

```yaml
info:
  app:
    name: Bulk Scanning
```

should be `Bulk Scan Processor`

that's what i mean - name should be checked. description i'm not so worry about right now

Reply:
changed the name